### PR TITLE
Redesigned Log In Page

### PR DESCRIPTION
**Pull request by jeffkim2001**
The new log in page has been created based on the new 2021 designs.

### CODE REVIEW

File: HackIllinois/UI/HIAppearance.swift [hunk 1]
```diff
@@ -126,7 +131,7 @@ struct HIAppearance: Equatable {
         static let navigationTitle = UIFont.systemFont(ofSize: 15, weight: .medium)
         static let descriptionText = UIFont.systemFont(ofSize: 14, weight: .regular)
         static let sponsorText = UIFont.systemFont(ofSize: 16, weight: .medium)
-        static let loginTitle = UIFont.systemFont(ofSize: 24, weight: .bold)
```
Comment (PatrickKan) [inline HackIllinois/UI/HIAppearance.swift:129]: Can it be bold to match the logo?

**Review by PatrickKan — COMMENTED** [review]
what's the diff between Login@ images and hackLogo?

**Comment by jeffkim2001** [comment]
Log In Page on iPhone SE
log-in-page-1

Log In Page on iPhone 11 Pro Max
log-in-page-2

**Review by PatrickKan — APPROVED** [review]
Looks good, thanks for the updates

**Pull request merged**